### PR TITLE
JENA-1562: Fix for Graph.size() for TDB2 graphs

### PR DESCRIPTION
**Pull request by afs**

### CODE REVIEW

File: jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/store/GraphViewSwitchable.java [hunk 1]
```diff
@@ -79,16 +83,34 @@ public TransactionHandler getTransactionHandler() {
...
+        if ( isDefaultGraph() )
+            return getDSG().getDefaultGraphTDB().size();
+        return getDSG().getGraphTDB(getGraphName()).size();
+    }
+
+    private DatasetGraphTDB getDSG() {
+        return ((DatasetGraphTDB)(getx().get()));    
+    }
+    
+    private static Function<Tuple<NodeId>, Tuple<NodeId>> project4TupleTo3Tuple = item -> {
```
Comment (ajs6f) [inline jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/store/GraphViewSwitchable.java]: This seems kind of out of place...

File: jena-arq/src/main/java/org/apache/jena/sparql/core/GraphView.java [hunk 2]
```diff
@@ -165,6 +165,9 @@ public void performDelete( Triple t ) {
         Node o = t.getObject() ;
         dsg.delete(g, s, p, o) ;
     }
+    
+    // Subclasses may wish to provide graphBaseSize otherwise GraphBase uses find()
```
Comment (ajs6f) [inline jena-arq/src/main/java/org/apache/jena/sparql/core/GraphView.java]: Maybe this is just where I like to find things, but I'd rather `@Override protected int graphBaseSize() { return super.graphBaseSize(); }` with this as a Javadoc comment.

Comment (afs) [reply jena-arq/src/main/java/org/apache/jena/sparql/core/GraphView.java]: Done.

File: jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/store/GraphViewSwitchable.java (hunk 1 above)
Comment (afs) [reply jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/store/GraphViewSwitchable.java]: Don't understand.  It is used in the method above and pulled out for clarity. It makes the handling of union graph clearer IMO with a named function at that point, not inline code.

Comment (ajs6f) [reply jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/store/GraphViewSwitchable.java]: I'm totally confused. The method above reads:
```
private DatasetGraphTDB getDSG() {
        return ((DatasetGraphTDB)(getx().get()));    
}
```
Where is `project4TupleTo3Tuple` used there?

Comment (afs) [reply jena-db/jena-tdb2/src/main/java/org/apache/jena/tdb2/store/GraphViewSwitchable.java]: OK - got it - it's a stray from GraphTDB. This one can be cleaned out.

**Pull request merged**